configuration/README.md

```diff
@@ -37,35 +49,50 @@ FieldTags:
   Val: source
 ```
 
-Sinks and sanitizers are identified via regexp according to package, method, and (optional) receiver name.
+Sinks and sanitizers are identified by package, method, and (optional) receiver name.
+Again, these may be specified by either a provided string literal or regexp.
 
 ```yaml
 Sinks:
-- PackageRE: <package path regexp>
+- Package:  "literal/package/path" 
   ReceiverRE: <type name regexp>
   MethodRE: <method name regexp>
 Sanitizers:
 - PackageRE: <package path regexp>
   ReceiverRE: <type name regexp>
-  MethodRE: <method name regexp>
+  Method: "mySanitizer"
 ```
 
 Taint propagation is performed automatically and does not need to be explicitly configured.
 
-For matchers that accept a `ReceiverRE` regexp matcher, an unspecified string will match any (or no) receiver.
-To match only methods without any receiver (i.e., a top-level function), use the matcher `^$` to match an empty-string receiver name.
+### Important Note
+
+Configuration of the above matchers does not require listing of all arguments.
+E.g., the following is a valid source configuration:
+```yaml
+Sources:
+- Package: "literal/package/path"
+  Type: "MySourceType"
+```
+
+Neither `Field` nor `FieldRE` have been provided.
+In this case, we match all fields of `MySourceType`, assuming a wildcard matcher for `FieldRE`.
+Similar behavior exists in for all attributes, e.g. providing neither `Type` nor `TypeRE` will match all type names.
+
+To explicitly match an empty string, such as top-level functions without a receiver, explicitly define the attribute by the corresponding string literal, `Receiver: ""`, or an anchored regexp, `ReceiverRE: "^$"`.
 
 ### Restricting analysis scope
 
-Functions can be explicitly excluded from analysis using regexps,
+Functions can be explicitly excluded from analysis using string literals or regexps,
 constructed similarly to those used to identify sanitizers and sinks:
 ```yaml
```

Comment:
I think this example + the associated explanations are great. I think the examples for Sources and Sinks/Sanitizers should be like this.